modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequestDetailPage.testcase

```diff
@@ -129,3 +129,35 @@ definition {
                 statusValue = "Approved");
         }
     }
+
+    @description = "This is a test for LPS-181999. Verify that when a MDf is approved, the activity status is Approved and Unclaimed"
+    @priority = 4
+    test ViewActivityStatusWhenMDFIsApproved {
+        task ("Given the Manager user is logged in and goes to MDF listing page") {
+            User.logoutAndLoginPG(
+                userLoginEmailAddress = "manager@liferay.com",
+                userLoginFullName = "manager");
+
+            PRMNavigator.openSitePage(pageName = "MDF Requests");
+        }
+        
+        task("When the manager approves the created MDF"){
+            PRMMDFRequest.goToMDF(campaignName = "Campaign Name 1");
+
+            PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+        }
+
+        task ("Then the manager can verify the activity status on the MDF detail page") {
```

Comment:
Hi @marinaasant0s, I saw that in the previous test you do the verification with the Partner user, but here you do the verification with the manager user. It's a little bit confused. What do you think about do all these validations with the Partner User?